guides/common/modules/proc_provisioning-ubuntu-autoinstall-through-smart-proxies.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Provisioning_Ubuntu_Autoinstall_Through_Smart_Proxies_{context}"]
+= Provisioning Ubuntu Autoinstall Through {SmartProxies}
+
+Perform these steps to provision hosts running Ubuntu 20.04.3+ or Ubuntu 22.04.
+
+.Procedure
+. On your {SmartProxyServer}, ensure _Smart Proxy HTTP forward_ is enabled:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# grep "^:enabled: true$" /etc/foreman-proxy/settings.d/templates.yml
+----
++
+Note that this is enabled by default.
+. On your {SmartProxyServer}, open the corresponding port:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# firewall-cmd --permanent --zone=public --add-port=8000/tcp
+# firewall-cmd --reload
+----
+. On your {ProjectServer}, add your {SmartProxyServer} to the list of trusted proxies:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-trusted-proxies "127.0.0.1" \
+--foreman-trusted-proxies "::1" \
+--foreman-trusted-proxies "_Your_{SmartProxyServer}_IP_"
```

Comment:
The underscores are not working well together with the value of the variable, which contains spaces. Maybe use spaces instead, and maybe rephrase, like: `"_IP of your {SmartProxyServer}_"` ? (The most important info first, but it doesn't fit the convetion for replaceable values we use elsewhere...)
Or just leave out the variable and say something like `"_proxy_IP_"`?

Reply:
Applied!